torch/_dynamo/guards.py

```diff
@@ -456,6 +457,10 @@ def CONSTANT_MATCH(self, guard: Guard):
         else:
             self.EQUALS_MATCH(guard)
 
+    def FULLGRAPH(self, guard: Guard):
+        code = ["___needs_nopython()"]
+        self._produce_guard_code(guard, code)
```

Comment:
This does not work as `eval_frame.guarded_backend_cache.nopython` does not exist anymore.

What would a good guard here? I tried `self.dynamo_ctx.nopython`